charts/harvester-network-controller/values.yaml

```diff
@@ -32,7 +32,7 @@ affinity: {}
 
 # Default value for harvester-network-controller-manager deployment
 manager:
-  replicas: 2
+  replicas: 1
```

Comment:
Why do we scale down to 1

Reply:
we may leverage the Rancher scale feature of single-node case https://github.com/harvester/harvester/blob/0345caa981570fcad6e15401b33656f6ad5d2bda/deploy/charts/harvester/templates/deployment.yaml#L6-L24

Reply:
We should keep the 2 replicas even with the annotation change